erp-data-exchange-specification.md

```diff
@@ -261,6 +262,29 @@ insert,07955b8c-41ac-4a47-9157-3c6fb8450ef4,9b86a4d7-ab95-4b65-a553-24fac1c60627
 insert,07955b8c-41ac-4a47-9157-3c6fb8450ef4,9b86a4d7-ab95-4b65-a553-24fac1c60627
 ```
 
+### responsibilityChannels.csv
+
+The `responsibilityChannels.csv` file describes the relation between an agent and a resource channel
+
+#### Fields
+
+| Field            | Type                            | Description                                                                                |
+| ---------------- | ------------------------------- | ------------------------------------------------------------------------------------------ |
+| **importType**   | [Import Type](#import-type)     | One of:<br/>`insert`, or `delete`                                                          |
+| **agentId**      | [UUID](#uuid-type)              | The foreign UUID of the agent that is responsible for the channel path (_agents.csv_ `id`) |
+| **resourceId**   | [UUID](#uuid-type)              | The foreign UUID of the resource that the agent belongs to ([resource].csv)                |
+| **resourceType** | [Resource Type](#resource-type) | The type of the resource being referenced by the resourceId (`property/group/unit`)        |
+
+#### Example
+
+```csv
+importType,agentId,resourceId,resourceType
+insert,07955b8c-41ac-4a47-9157-3c6fb8450ef4,ab463b8b-a76c-4f6a-a726-75ab5730b69b, property
```

Comment:
maybe just a stupid question, but once you call the api the uuid (externalId) is converted to our id via the id-lookup right?

Reply:
ya